tests/gateway/test_braintree.py

```diff
@@ -263,250 +255,162 @@ def test_authorize_error_response(
     assert response.error == DEFAULT_ERROR
 
 
-@pytest.mark.integration
-@patch("saleor.payment.gateways.braintree.get_braintree_gateway")
-def test_authorize_incorrect_token(
-    mock_gateway, payment_dummy, braintree_not_found_error, gateway_config
-):
-    payment = payment_dummy
-    payment_token = "payment-token"
-    mock_response = Mock(side_effect=braintree_not_found_error)
-    mock_gateway.return_value = Mock(transaction=Mock(sale=mock_response))
-
-    payment_info = create_payment_information(payment, payment_token)
-    with pytest.raises(BraintreeException) as e:
-        authorize(payment_info, gateway_config)
-    assert str(e.value) == DEFAULT_ERROR_MESSAGE
+@pytest.fixture
+def sandbox_braintree_gateway_config(gateway_config):
+    """ set up your environment variables to record sandbox """
+    gateway_config.connection_params = {
+        "merchant_id": os.getenv("BRAINTREE_MERCHANT_ID", "9m6qhfxsqzm3cgzw"),
```

Comment:
Why trying to get them from the environment?

Reply:
To integrate gateway you need to set those up either way so it is probably best way to leave it to env for next person to record cassettes. We *surely* do not want those in code.